components/admin_console/custom_enable_disable_guest_accounts_setting.tsx

```diff
@@ -13,7 +13,7 @@ import BooleanSetting from './boolean_setting';
 type Props = {
     id: string;
     value: boolean;
-    onChange: (id: string, value: any) => void;
+    onChange: (id: string, value: any, submit: boolean) => void;
```

Comment:
I think we would want to change the name of this too.
```suggestion
    onChange: (id: string, value: any, submitImmediately: boolean) => void;
```